docs/troubleshooting.md

```diff
@@ -51,7 +51,7 @@ interpreter accordingly (in most cases open the path and look for
 poetry env info -p
 ```
 
-### VSCode Python extension no longer supports `python.pythonPath` in `setting.json`
+### VSCode Python extension no longer supports \`python.pythonPath\` in \`setting.json\`
```

Comment:
The \` is used to mark the text as code. If you want to quote something, you can use the ", not \`.

Reply:
Got it. Changed to the double quotation mark.